app/graphql/types/query_type.rb

```diff
@@ -44,26 +44,35 @@ def record_id(id:, index:)
-        argument :subjects, String, required: false, default_value: nil
-        argument :title, String, required: false, default_value: nil
-        argument :from, String, required: false, default_value: '0'
+        argument :searchterm, String, required: false, default_value: nil, description: 'Query all searchable fields'
+        argument :citation, String, required: false, default_value: nil, description: 'Search by citation information'
+        argument :contributors, String, required: false, default_value: nil, description: 'Search by contributor names'
+        argument :funding_information, String, required: false, default_value: nil,
+                                               description: 'Search by funding information; e.g., funding source, award name, etc.'
+        argument :identifiers, String, required: false, default_value: nil,
+                                       description: 'Search by unique indentifier; e.g., ISBN, DOI, etc.'
+        argument :locations, String, required: false, default_value: nil, description: 'Search by locations'
+        argument :subjects, String, required: false, default_value: nil, description: 'Search by subject terms'
+        argument :title, String, required: false, default_value: nil, description: 'Search by title'
+        argument :from, String, required: false, default_value: '0',
+                                description: 'Search result number to begin with (the first result is 0)'
         argument :index, String, required: false, default_value: nil,
                                  description: 'It is not recommended to provide an index value unless we have provided you with one for your specific use case'
 
         # applied facets
-        argument :content_type_facet, [String], required: false, default_value: nil
-        argument :contributors_facet, [String], required: false, default_value: nil
-        argument :format_facet, [String], required: false, default_value: nil
-        argument :languages_facet, [String], required: false, default_value: nil
-        argument :literary_form_facet, String, required: false, default_value: nil
-        argument :source_facet, [String], required: false, default_value: nil
-        argument :subjects_facet, [String], required: false, default_value: nil
+        argument :content_type_facet, [String], required: false, default_value: nil,
+                                                description: 'Filter results by content type'
+        argument :contributors_facet, [String], required: false, default_value: nil,
+                                                description: 'Filter results by contributors'
+        argument :format_facet, [String], required: false, default_value: nil,
+                                          description: 'Filter results by format; e.g., "Print Volume", "DVD", etc.'
+        argument :languages_facet, [String], required: false, default_value: nil,
```

Comment:
For all of these _facet fields, I wonder if there is a way to hint that the best way to get these values is from the corresponding aggregation. I don't have a specific suggestion on how to do that, so I'd be okay with just thinking about that as a future enhancement.

Reply:
Might be good to include in how-to/reference docs? That said, I guess these very docs could fall into either of those categories. I'm not opposed to adding a note to that effect to each of the facet fields.

Reply:
Good point. I like the idea of both including an example of how this works in our docs and linking to that example from here once written.

Reply:
Lol yeah, after commenting I realized I could cut down the word count substantially. English major tendencies. :)